### PR TITLE
Agents august updates

### PR DESCRIPTION
**Pull request by LaurentLesle**
Update versions
Add node 16 to azdo (still not released yet to docker hub see Issue #280 )
Add kubernetes autoscale support for gihub agent

### CODE REVIEW

File: .env.terraform [hunk 1]
```diff
@@ -1,3 +1,3 @@
-1.2.5
+1.2.7
```
Comment (arnaudlh) [inline .env.terraform:1]:
```suggestion
1.2.9
```

File: .env.terraform [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 1.1.9
-1.3.0-alpha20220706
+1.3.0-alpha20220803
```
Comment (arnaudlh) [inline .env.terraform:3]:
```suggestion
1.3.0-beta1
```

**Review by arnaudlh — CHANGES_REQUESTED** [review]
Updates to TF

File: .env.terraform [hunk 3]
```diff
@@ -1,3 +1,3 @@
-1.2.5
+1.2.9
 1.1.9
-1.3.0-alpha20220706
+1.3.0-beta1
```
Comment (arnaudlh) [inline .env.terraform:3]:
```suggestion
1.3.0-rc1
```

**Review by arnaudlh — APPROVED** [review]
LGTM

**Comment by dsfrederic** [comment]
Why are the azdo agents not released

**Pull request merged**